modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -397,4 +570,367 @@ If `LOAD CSV` is invoked with a `file:///` URL that points to your disk `file()`
+----
+Id,Name,Year
+1,ABBA,1992
+2,Roxette,1986
+3,Europe,1979
+4,The Cardigans,1992
+----
+
+.Query
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'file:///artists-with-headers.csv' AS row
+MERGE (:Artist {name: row.Name, year: toInteger(row.Year)})
+----
+
+.Result
+[source, role="queryresult"]
+----
++-------------------+
+| No data returned. |
++-------------------+
+Nodes created: 4
+Properties set: 8
+Labels added: 4
+----
+====
+
+
+=== Field delimiter
+
+The default field delimiter is `,`.
+Use the `FIELDTERMINATOR` option to specify a different field delimiter.
+
+If you try to import a file that doesn't use `,` as field delimiter and you also don't specify a custom delimiter, `LOAD CSV` will interpret the CSV as having a single column.
+
+.Import a CSV using `;` as field delimiter
+====
+.artists-fieldterminator.csv
+[source, csv, filename="artists-fieldterminator.csv"]
+----
+1;ABBA;1992
+2;Roxette;1986
+3;Europe;1979
+4;The Cardigans;1992
+----
+
+.Query
+[source, cypher]
+----
+LOAD CSV FROM 'file:///artists-fieldterminator.csv' AS row FIELDTERMINATOR ';'
+MERGE (:Artist {name: row[1], year: toInteger(row[2])})
+----
+
+.Result
+[source, role="queryresult"]
+----
++-------------------+
+| No data returned. |
++-------------------+
+Nodes created: 4
+Properties set: 8
+Labels added: 4
+----
+====
+
+
+[NOTE]
+You can use the hexadecimal representation of the unicode character for the field delimiter if you prepend `{backslash}u`.
+Write the encoding with four digits: for example, `{backslash}u003B` is equivalent to `;` (semicolon).
+
+
+=== Quotes escaping
+
+Quoted strings are allowed in the CSV file and the quotes are dropped when `LOAD CSV` reads the data.
+If quoted strings need to contain quote characters `"`, there are two ways to escape them:
+
+1. *Double quotes* -- Use another quote `"` to escape a quote (for example, the CSV encoding of the string `The "Symbol"` is `"The ""Symbol"""`).
+2. *Prefix with backslash `\`* -- If the configuration setting `dbms.import.csv.legacy_quote_escaping` is set to `true` (the default value), `\` works as the escape character for quotes (for example, the CSV encoding of the string `The "Symbol"` is `"The {backslash}"Symbol{backslash}""`).
+
+.Import a CSV with double-quotes escaping
+====
+.artists-with-escaped-quotes.csv
+[source, csv, filename="artists-with-escaped-quotes.csv"]
+----
+"1","The ""Symbol""","1992"
+"1","The \"Symbol\"","1992"
+----
+
+.Query
+[source, cypher]
+----
+LOAD CSV FROM 'file:///artists-with-escaped-quotes.csv' AS row
+MERGE (a:Artist {name: row[1], year: toInteger(row[2])})
+RETURN
+  a.name AS name,
+  a.year AS year,
+  size(a.name) AS size
+----
+
+.Result
+[source, role="queryresult",options="header,footer",cols="3*<m"]
+|===
+| name | year | size
+| 'The "Symbol"' | 1992 | 12
+| 'The "Symbol"' | 1992 | 12
+3+d| Nodes created: 2 +
+Properties set: 4 +
+Labels added: 1
+|===
+
+Note that `name` is a string, as it is wrapped in single quotes in the output.
```

Comment:
Whether it is wrapped in single or double quotes is interface dependent, so saying it is a string because it is wrapped in single quotes isn't 100% true.

Reply:
rephrased